content/webapp/pages/stories.tsx

```diff
@@ -85,7 +85,7 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     const articlesPromise = getArticles(context.req, { page }, memoizedPrismic);
     const seriesPromise = getArticleSeries(
       context.req,
-      { id: 'YRpHthEAAJoN3wfc' },
+      { id: 'YXKNnxEAACEARPrl' },
```

Comment:
Is it worth a comment about what this is doing? i.e. that this is the main page highlighted on the Stories homepage

(Speaking as somebody who wouldn't have been sure where to find this or change it.)

Reply:
Yep, definitely. For the longest time this is something that we've wanted to make cms-able, but there hasn't been the bandwidth. I'll add the comment.